.circleci/config.yml

```diff
@@ -652,6 +653,8 @@ workflows:
           environment: development
           image_name: tariff-backend
           ssm_parameter: "/development/BACKEND_ECR_URL"
+          requires:
```

Comment:
This isn't needed - since it's blocked at the plan-terraform stage and would mean build and cannot happen concurrently with the rubocop/brakeman checks

Reply:
Ok i only included this not to waste a tiny bit of processing if it doesnt pass
Happy to remove